app/models/task.rb

```diff
@@ -0,0 +1,5 @@
+class Task < ApplicationRecord
+  validates :name, presence: true, uniqueness: true
+  # ユニーク制約は一応貼っておく
+  # TODO DB上でユニークが設定出来てるか不明瞭
```

Comment:
[nits]
直さなくていいレベルだけど、アノテーションはコロンもつけた方がよさげ
```suggestion
  # TODO: DB上でユニークが設定出来てるか不明瞭
```

参考
[rubocop\-hq/ruby\-style\-guide: A community\-driven Ruby coding style guide](https://github.com/rubocop-hq/ruby-style-guide#annotations-keyword-format)

Reply:
e177d2675391ab23ec5fabdbd9a3bad816d2d4f9  で修正しました！